build fail looks real
```

======================================================================
ERROR: test_cases_owned_by_group (corehq.apps.callcenter.tests.test_indicators.CallCenterCaseSharingTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/callcenter/tests/test_indicators.py", line 481, in test_cases_owned_by_group
    custom_cache=locmem_cache
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/callcenter/indicator_sets.py", line 88, in __init__
    self.config = indicator_config or CallCenterIndicatorConfig.for_domain(domain_name)
  File "/home/travis/build/dimagi/commcare-hq/corehq/util/quickcache.py", line 308, in inner
    return helper(*args, **kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/util/quickcache.py", line 200, in __call__
    content = self.fn(*args, **kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/callcenter/models.py", line 56, in for_domain
    res = get_docs_in_domain_by_class(domain, cls)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/domain/dbaccessors.py", line 43, in get_docs_in_domain_by_class
    include_docs=True,
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/couchdbkit/client.py", line 1014, in all
    return list(self.iterator())
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/couchdbkit/client.py", line 973, in iterator
    self._fetch_if_needed()
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/couchdbkit/client.py", line 1052, in _fetch_if_needed
    self.fetch()
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/couchdbkit/client.py", line 1031, in fetch
    self._result_cache = self.fetch_raw().json_body
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/couchdbkit/client.py", line 1048, in fetch_raw
    return self._fetch(self._arg, self.params)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/couchdbkit/client.py", line 723, in raw_view
    return self.res.get(view_path, **params)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/restkit/resource.py", line 114, in get
    params_dict=params_dict, **params)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/couchdbkit/resource.py", line 136, in request
    response=e.response)
ResourceNotFound: missing
```

--- a/corehq/apps/domain/__init__.py
+++ b/corehq/apps/domain/__init__.py
@@ -2,4 +2,4 @@ from corehq.preindex import ExtraPreindexPlugin
 from django.conf import settings
 
 ExtraPreindexPlugin.register('domain', __file__, (
-    settings.NEW_USERS_GROUPS_DB, settings.NEW_FIXTURES_DB))
+    settings.NEW_USERS_GROUPS_DB, settings.NEW_FIXTURES_DB, 'meta'))

--- a/corehq/couchapps/tests/test_all_docs.py
+++ b/corehq/couchapps/tests/test_all_docs.py
@@ -31,7 +31,8 @@ class AllDocsTest(TestCase):
              get_all_doc_ids_for_domain_grouped_by_db(self.domain)},
             {get_db(None).uri: ['main_db_doc'],
              get_db('users').uri: ['users_db_doc'],
-             get_db('fixtures').uri: []}
+             get_db('fixtures').uri: [],
+             get_db('meta').uri: []}
         )
 
     def test_get_doc_count_by_type(self):

